core/src/main/scala/kafka/server/ControllerApis.scala

```diff
@@ -410,6 +409,12 @@ class ControllerApis(val requestChannel: RequestChannel,
         } else {
           new ApiError(TOPIC_AUTHORIZATION_FAILED)
         }
+      case ConfigResource.Type.CLIENT_METRICS =>
```

Comment:
Do we really need such granular permissions for this? I would have thought `ALTER_CONFIGS` on `CLUSTER` would be good enough.

It seems really odd to set up a framework for allowing or denying people permission to change individual client metrics. What's the use case?

Reply:
I am bit confused here,  do you mean to say that no need for the CLIENT_METRICS resource at all  OR  just use the CLUSTER authorization for CLIENT_METRICS resource too..

FYI: I have added new resource type CLIENT_METRICS based on the proposal in KIP-714: 
https://cwiki.apache.org/confluence/display/KAFKA/KIP-714%3A+Client+metrics+and+observability#KIP714:Clientmetricsandobservability-Metricssubscription